### PR TITLE
feat: add etcd client rate limit

### PR DESCRIPTION
**Pull request by zhu327**
### Description

Fixes # (issue)

### Checklist

- [ ] 填写 PR 描述及相关 issue (write PR description and related issue)
- [ ] 代码风格检查通过 (code style check passed)
- [ ] PR 中包含单元测试 (include unit test)
- [ ] 单元测试通过 (unit test passed)
- [ ] 本地开发联调环境验证通过 (local development environment verification passed)

### CODE REVIEW

File: pkg/config/config.go [hunk 1]
```diff
@@ -274,6 +276,7 @@ func newDefaultConfig() *Config {
 			AgentCommitTimeWindow:        5 * time.Second,
 			AgentConcurrencyLimit:        2,
 			EtcdPutInterval:              50 * time.Millisecond,
+			EtcdRateLimit:                100,
```
Comment (wklken) [inline pkg/config/config.go:279]: 改成一个对象, 同时支持配置 limits和bursts

File: pkg/apisix/synchronizer/etcd/store.go [hunk 2]
```diff
@@ -266,6 +291,11 @@ func (s *EtcdConfigStore) batchPutResource(ctx context.Context, resourceType str
 
 		s.logger.Debugw("Put resource to etcd", "resourceType", resourceType, "resourceID", resource.GetID())
 
+		// client操作限流
+		if needRateLimit {
```
Comment (wklken) [inline pkg/apisix/synchronizer/etcd/store.go:295]: 增加一个单测, 验证limit生效(需要看下好不好加, 因为这里是wait, 需要有方法assert只有一个放通)

File: pkg/config/config.go (hunk 1 above)
Comment (zhu327) [reply pkg/config/config.go]: Done

**Comment by wklken** [comment]
需要其他方式解决

**Pull request closed without merging**